src/rws_common.cpp

```diff
@@ -165,18 +165,19 @@ const std::string SystemConstants::ContollerStates::CONTROLLER_MOTOR_OFF      =
 const std::string SystemConstants::ContollerStates::PANEL_OPERATION_MODE_AUTO = "AUTO";
 const std::string SystemConstants::ContollerStates::RAPID_EXECUTION_RUNNING   = "running";
 
-const std::string SystemConstants::General::EXTERNAL_APPLICATION  = "ExternalApplication";
-const std::string SystemConstants::General::EXTERNAL_LOCATION     = "ExternalLocation";
-const std::string SystemConstants::General::DEFAULT_PASSWORD      = "robotics";
-const std::string SystemConstants::General::DEFAULT_USERNAME      = "Default User";
-const std::string SystemConstants::General::LOCAL                 = "local";
-const std::string SystemConstants::General::MECHANICAL_UNIT_ROB_1 = "ROB_1";
-const std::string SystemConstants::General::MECHANICAL_UNIT_ROB_2 = "ROB_2";
-const std::string SystemConstants::General::MECHANICAL_UNIT_ROB_3 = "ROB_3";
-const std::string SystemConstants::General::MECHANICAL_UNIT_ROB_4 = "ROB_4";
-const std::string SystemConstants::General::MECHANICAL_UNIT_ROB_L = "ROB_L";
-const std::string SystemConstants::General::MECHANICAL_UNIT_ROB_R = "ROB_R";
-const std::string SystemConstants::General::REMOTE                = "remote";
+const std::string SystemConstants::General::EXTERNAL_APPLICATION   = "ExternalApplication";
+const std::string SystemConstants::General::EXTERNAL_LOCATION      = "ExternalLocation";
+const unsigned short SystemConstants::General::DEFAULT_PORT_NUMBER = 80;
+const std::string SystemConstants::General::DEFAULT_PASSWORD       = "robotics";
+const std::string SystemConstants::General::DEFAULT_USERNAME       = "Default User";
+const std::string SystemConstants::General::LOCAL                  = "local";
+const std::string SystemConstants::General::MECHANICAL_UNIT_ROB_1  = "ROB_1";
+const std::string SystemConstants::General::MECHANICAL_UNIT_ROB_2  = "ROB_2";
+const std::string SystemConstants::General::MECHANICAL_UNIT_ROB_3  = "ROB_3";
+const std::string SystemConstants::General::MECHANICAL_UNIT_ROB_4  = "ROB_4";
+const std::string SystemConstants::General::MECHANICAL_UNIT_ROB_L  = "ROB_L";
+const std::string SystemConstants::General::MECHANICAL_UNIT_ROB_R  = "ROB_R";
+const std::string SystemConstants::General::REMOTE                 = "remote";
```

Comment:
We should probably change some of this to `constexpr`s (but that would need C++11 again).

Reply:
Ok, I will keep it in mind.